src/commands/lambda/functions/commons.ts

```diff
@@ -258,12 +260,12 @@ export const getLambdaFunctionConfigs = (
  */
 export const getLayerArn = (
   config: Lambda.FunctionConfiguration,
-  runtime: Runtime,
+  layer: LayerName,
   region: string,
   settings?: InstrumentationSettings
 ) => {
-  let layerName = RUNTIME_LAYER_LOOKUP[runtime]
-  if (ARM_RUNTIMES.includes(runtime) && config.Architectures?.includes(ARM64_ARCHITECTURE)) {
+  let layerName = LAYER_LOOKUP[layer]
```

Comment:
Here is a good example of why I think it would be better to name the type `LayerName` as `LayerKey`.

Here `layer` is a `LayerName`. But, it's not actually the name of a layer. To get the name of the layer, we need to look it up with LAYER_LOOKUP.

If `layer` were a `LayerKey`, it would make more sense that we use it to GET the name of the layer.